build.sh

```diff
@@ -216,25 +212,19 @@ isMSBuildOnNETCoreSupported()
 
     if [ "$__BuildArch" == "x64" ]; then
         if [ "$__BuildOS" == "Linux" ]; then
-            if [ "$__DistroName" == "ubuntu" ]; then
-                __OSVersion=$(lsb_release -rs)
-                if [ "$__OSVersion" == "14.04" ]; then
-                    __isMSBuildOnNETCoreSupported=1
-                elif [ "$(cat /etc/*-release | grep -cim1 14.04)" -eq 1 ]; then
-                    # Linux Mint based on Ubuntu 14.04
-                    __isMSBuildOnNETCoreSupported=1
-                fi
-            elif [ "$__DistroName" == "rhel" ]; then
+            if [ "$__DistroRid" == "ubuntu.14.04-x64" ]; then
```

Comment:
I am going to change this whole thing completely soon.  Right now we don't have 16.04 CLIs.